packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -251,6 +251,9 @@ export class OdspDocumentService implements IDocumentService {
      * @returns returns the document delta stream service for onedrive/sharepoint driver.
      */
     public async connectToDeltaStream(client: IClient): Promise<IDocumentDeltaConnection> {
+        if (this.currentConnection !== undefined && !this.currentConnection.disposed) {
```

Comment:
Is there actual scenario where we think it happens and that's the right flow?
If not, I'd rather not allow it (i.e. assert here that this.currentConnection === undefined) and add it only when we have a scenario to support.
Basically, in a current system we assume one user for delta stream, and there are probably various assumptions through the code based on that. When we reuse connection, it's possible we are breaking these assumptions without knowing about it.

Reply:
No according to our flows, we just connect one time. Maybe we should not add anything because then currentConnection needs to set to undefined somewhere. I would not like to add an assert either. So maybe just leave it as such without any change? We haven't had any issue related to this.

Reply:
Curious, why you do not like to add assert?
I do not like leave it as is. Either we make a decision that we support only one object at a time (then assert should be added). Or we declare that we support multiple calls and ensure this works correctly and is fully tested.